src/common/contexts/themeContext/themes.ts

```diff
@@ -0,0 +1,17 @@
+export interface ThemeProps {
+  background: string;
+  text: string;
+  primary: string;
+}
+
+export const darkTheme: ThemeProps = {
+  background: 'var(--dark-background)',
```

Comment:
Instead of declaring our variables as global, we just insert our theme styles here